app/controllers/wiki_controller.rb

```diff
@@ -119,10 +119,10 @@ def edit
 
   def new
     @node = Node.new
-    if params[:n] && !params[:body] # use another node body as a template
-      node = Node.find(params[:n])
-      params[:body] = node.latest.body if node&.latest
-    end
+    # if params[:n] && !params[:body] # use another node body as a template
+    #   node = Node.find(params[:n])
+    #   params[:body] = node.latest.body if node&.latest
```

Comment:
see how this uses `node.latest.body` but the lower one uses `node.body` ? I think the top one is strictly more accurate, with `latest`.